features/zoek-met-straat-huisnummer-gemeentevaninschrijving.feature

```diff
@@ -0,0 +1,345 @@
+    | 999994402           | Vitternkade              |                              | 101                |                    | II                           |
+    | 999993008           | Afrikanerplein           |                              | 1                  |                    |                              |
+    | 999993009           | Afrikanerplein           |                              | 1                  |                    |                              |
+    | 999990159           | Ln Copes van Cattenburch | Laan Copes van Cattenburch   | 38                 |                    |                              |
+    En het systeem heeft personen met de volgende 'overlijden' gegevens
+    | burgerservicenummer | datum overlijden (08.10) |
+    | 999993008           | 20080700                 |
+
+  Rule: Straat, huisnummer en gemeente van inschrijving zijn verplichte parameters
+        - straat parameter zoekt over zowel straatnaam als naam openbare ruimte
+        - zoeken is niet hoofdlettergevoelig
+  
+    Abstract Scenario: Zoek met volledige straat, huisnummer en gemeente van inschrijving
+      Als personen wordt gezocht met de volgende parameters
+      | naam                    | waarde                                           |
+      | type                    | ZoekMetStraatHuisnummerEnGemeenteVanInschrijving |
+      | straat                  | <straat>                                         |
+      | huisnummer              | 31                                               |
+      | gemeenteVanInschrijving | 0599                                             |
+      | fields                  | burgerservicenummer                              |
+      Dan heeft de response 1 persoon
+      En heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999993653 |
+
+      Voorbeelden:
+      | straat    |
+      | Boterdiep |
+      | boterdiep |
+      | BOTERDIEP |
+
+    Scenario: Zoek met straat van één karakter lang
+      Als personen wordt gezocht met de volgende parameters
+      | naam                    | waarde                                           |
+      | type                    | ZoekMetStraatHuisnummerEnGemeenteVanInschrijving |
+      | straat                  | a                                                |
+      | huisnummer              | 13                                               |
+      | gemeenteVanInschrijving | 0693                                             |
+      | fields                  | burgerservicenummer                              |
+      Dan heeft de response 1 persoon
+      En heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999993914 |
+
+    Scenario: Zoek met naam openbare ruimte
```

Comment:
Zoek met huisnummer, gemeente van inschrijving en straat waarbij straat gelijk is aan de openbare ruimtenaam